deployment/7_upgrade_2_0.js

```diff
@@ -49,26 +66,23 @@ const deploy = async (network) => {
   const ModuleRegistryWrapper = await deployer.wrapDeployedContract(ModuleRegistry, config.contracts.ModuleRegistry);
   const MultiSigWrapper = await deployer.wrapDeployedContract(MultiSig, config.contracts.MultiSigWallet);
   const multisigExecutor = new MultisigExecutor(MultiSigWrapper, deploymentWallet, config.multisig.autosign, { gasPrice });
+  const ENSManagerWrapper = await deployer.wrapDeployedContract(ENSManager, config.contracts.ENSManager);
 
   // //////////////////////////////////
   // Deploy infrastructure contracts
   // //////////////////////////////////
 
-  // Deploy and configure Maker Registry
-  const ScdMcdMigrationWrapper = await deployer.wrapDeployedContract(ScdMcdMigration, config.defi.maker.migration);
-  const vatAddress = await ScdMcdMigrationWrapper.vat();
-  const MakerRegistryWrapper = await deployer.deploy(MakerRegistry, {}, vatAddress);
-  const wethJoinAddress = await ScdMcdMigrationWrapper.wethJoin();
-  const addCollateralTransaction = await MakerRegistryWrapper.contract.addCollateral(wethJoinAddress, { gasPrice });
-  await MakerRegistryWrapper.verboseWaitForTransaction(addCollateralTransaction, `Adding join adapter ${wethJoinAddress} to the MakerRegistry`);
-  const changeMakerRegistryOwnerTx = await MakerRegistryWrapper.contract.changeOwner(config.contracts.MultiSigWallet, { gasPrice });
-  await MakerRegistryWrapper.verboseWaitForTransaction(changeMakerRegistryOwnerTx, "Set the MultiSig as the owner of the MakerRegistry");
-  const TokenPriceProviderWrapper = await deployer.wrapDeployedContract(TokenPriceProvider, config.contracts.TokenPriceProvider);
+  // Deploy the Base Wallet Library
+  const BaseWalletWrapper = await deployer.deploy(BaseWallet);
+  // Deploy TokenPriceProvider
+  const TokenPriceProviderWrapper = await deployer.deploy(TokenPriceProvider);
+  // Deploy the Wallet Factory
+  const WalletFactoryWrapper = await deployer.deploy(WalletFactory, {},
+    ModuleRegistryWrapper.contractAddress, BaseWalletWrapper.contractAddress, ENSManagerWrapper.contractAddress);
```

Comment:
If we deploy a new Factory we need to add it as a Manager of the ENSManager.

Reply:
Also forgotten to add the backend accounts as managers of the Factory and TokenPriceProvider

Reply:
And set the final owner of the WalletFactory to MultiSig also